### PR TITLE
[SPARK-46586][SQL][TESTS][FOLLOWUP] Replace `appended` with `:+`

### PR DESCRIPTION
**Pull request by panbingkun**
### What changes were proposed in this pull request?
The pr is following up https://github.com/apache/spark/pull/44591.

### Why are the changes needed?
Fix issues in comments.
image

### Does this PR introduce _any_ user-facing change?
No.

### How was this patch tested?
- Pass GA.
- Manually test.

### Was this patch authored or co-authored using generative AI tooling?
No.

### CODE REVIEW

**Comment by panbingkun** [comment]
cc @cloud-fan

**Review by LuciferYang — APPROVED** [review]

**Review by cloud-fan — APPROVED** [review]

File: sql/core/src/test/scala/org/apache/spark/sql/UDFSuite.scala [hunk 1]
```diff
@@ -840,7 +840,7 @@ class UDFSuite extends QueryTest with SharedSparkSession {
     Row(ArrayBuffer(100)))
 
      val myUdf2 = udf((a: immutable.ArraySeq[Int]) =>
-      immutable.ArraySeq.unsafeWrapArray[Int](a.appended(5).appended(6).toArray))
+      immutable.ArraySeq.unsafeWrapArray[Int]((a :+ 5 :+ 6).toArray))
```
Comment (HyukjinKwon) [inline sql/core/src/test/scala/org/apache/spark/sql/UDFSuite.scala:843]: Can we avoid this kind of followups. and PRs. This is just syntactic sugar

**Comment by HyukjinKwon** [comment]
LGTM since the PR is already open tho

**Comment by LuciferYang** [comment]
Merged into master. Thanks @panbingkun @cloud-fan

**Pull request closed without merging**